catalogs/sources/gtfs/realtime/us-california-santa-cruz-metro-scmtd-gtfs-rt-t-u-1911.json

```
@@ -0,0 +1,13 @@
+{
+    "mdb_source_id": 1911,
+    "data_type": "gtfs-rt",
+    "entity_type": "tu",
```

Convert the entity_type to an array:
```suggestion
    "entity_type": ["tu"],
```